web/src/views/SubmissionPortal/Components/MetadataSuggester.vue

```diff
@@ -0,0 +1,500 @@
+                      v-text="s.current_value"
+                    />
+                    <span
+                      class="value suggested"
+                      v-text="s.value"
+                    />
+                  </div>
+
+                  <div class="flex-shrink-0 flex-grow-0">
+                    <v-tooltip
+                      bottom
+                      :open-delay="TOOLTIP_DELAY"
+                    >
+                      <template #activator="{ on, attrs }">
+                        <v-btn
+                          icon
+                          color="primary"
+                          v-bind="attrs"
+                          v-on="on"
+                          @click="handleJumpToCell(s)"
+                        >
+                          <v-icon>
+                            mdi-target
+                          </v-icon>
+                        </v-btn>
+                      </template>
+                      <span>Jump to cell</span>
+                    </v-tooltip>
+
+                    <v-tooltip
+                      bottom
+                      :open-delay="TOOLTIP_DELAY"
+                    >
+                      <template #activator="{ on, attrs }">
+                        <v-btn
+                          icon
+                          color="primary"
+                          v-bind="attrs"
+                          v-on="on"
+                          @click="handleRejectSuggestion(s)"
+                        >
+                          <v-icon>
+                            mdi-close
+                          </v-icon>
+                        </v-btn>
+                      </template>
+                      <span>Reject suggestion</span>
+                    </v-tooltip>
+
+                    <v-tooltip
+                      bottom
+                      :open-delay="TOOLTIP_DELAY"
+                    >
+                      <template #activator="{ on, attrs }">
+                        <v-btn
+                          icon
+                          color="primary"
+                          v-bind="attrs"
+                          v-on="on"
+                          @click="handleAcceptSuggestion(s)"
+                        >
+                          <v-icon>
+                            mdi-check
+                          </v-icon>
+                        </v-btn>
+                      </template>
+                      <span>Accept suggestion</span>
+                    </v-tooltip>
+                  </div>
+                </div>
+              </div>
+            </v-sheet>
+          </div>
+        </v-col>
+      </v-row>
+
+      <v-row v-if="rejectedSuggestions.length > 0">
+        <v-col>
+          <v-btn
+            class="text-no-transform"
+            color="grey darken-2"
+            block
+            plain
+            @click="handleResetRejectedSuggestions"
+          >
+            Reset rejected suggestions
+          </v-btn>
```

Comment:
Is there a reason that this button is `plain`? I would have expected to match the style of existing buttons on the page.

Reply:
Yeah I don't know 😂 Just trying to lower the visual priority of it, but ended up overcooking it. I've updated it so that it uses the same style as the "full reference" button in the Column Info tab.